Source/KerbalCombatSystems/Utils.cs

```diff
@@ -234,15 +237,12 @@ public static ModuleDecouplerDesignate FindDecoupler(Part origin, string type)
             return null;
         }
 
-        public static List<ModuleDecouplerDesignate> FindDecouplerChildren(Part root, string type)
+        // Search the children of a specified part for separators.
+        public static List<ModuleDecouplerDesignate> FindDecouplerChildren(Part root, string type = "Default")
         {
-            // method to search the children of a specified part for decoupler modules
-            bool defaultCoupler = (type != "" || type != null || type != "Default");
-
             List<Part> childParts = root.FindChildParts<Part>(true).ToList();
-            //check the parent itself
```

Comment:
//check the parent itself

keep this in, more documentation is always positive